include/deal.II/fe/mapping_manifold.h

```diff
@@ -434,6 +437,23 @@ MappingManifold<dim, spacedim>::preserves_vertex_locations() const
   return true;
 }
 
+
+template <int dim, int spacedim>
+bool
+MappingManifold<dim, spacedim>::is_compatible_with(
+  const ReferenceCell::Type &cell_type) const
```

Comment:
also to the `.cc` file. same for the comments I marked up above and that apply here as well.